src/libraries/Microsoft.Extensions.Configuration.Binder/src/Resources/Strings.resx

```diff
@@ -120,6 +120,12 @@
   <data name="Error_CannotActivateAbstractOrInterface" xml:space="preserve">
     <value>Cannot create instance of type '{0}' because it is either abstract or an interface.</value>
   </data>
+  <data name="Error_CannotBindToConstructorParameter" xml:space="preserve">
+    <value>Cannot create instance of type '{0}' because one or more parameters cannot be bound to. Constructor parameters cannot be declared as in, out, or ref. Invalid parameters are: '{1}'</value>
+  </data>
+  <data name="Error_ConstructorParametersDoNotMatchProperties" xml:space="preserve">
+    <value>Cannot create instance of type '{0}' because one or more parameters cannot be bound to. Constructor parameters must have corresponding properties. Fields are not supported.  Missing properties are: '{1}'</value>
```

Comment:
nit, spacing

```suggestion
    <value>Cannot create instance of type '{0}' because one or more parameters cannot be bound to. Constructor parameters must have corresponding properties. Fields are not supported. Missing properties are: '{1}'</value>
```